package.json

```diff
@@ -52,37 +95,34 @@
     },
     "devDependencies": {
         "@fluid-project/hyphenation-patterns": "0.2.2-dev.20181115T221631Z.b2d0651",
+        "bestzip": "2.2.0",
         "browserslist-config-fluid": "0.1.0",
+        "cpy-cli": "3.1.1",
```

Comment:
I don't see any use of this dependency in this pull

Reply:
it's used for the dependency copying, moving files from nodue_modules to the lib directory. However, the command is just `cpy` as this is just a wrapper package to add a CLI API.